modules/ROOT/pages/policies-policy-overview.adoc

```diff
@@ -0,0 +1,87 @@
+= Policies Overview
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+:keywords: policy, custom, ootb, offline
+
+Policies differ based on several different factors, such as category, purpose, version, and configuration options. You must carefully review these factors, before you decide to implementation them in your environment. In this section, each of these factors are explained in detail. 
+
+== Policy Architecture
+
+Policies are implemented through coordinated communication between the following components:
+
+* API Manager
+* One or more API Gateway Runtimes, or Mule 3.8.0 Runtime or later
+* One or more API proxy or API applications
+
+Using API Manager, you can configure and apply policies to an API instance. An API version can have one or more API instances. +
+A specific API instance can also be tied to a specific API implementation endpoint, which can then auto-generate an API proxy application. +
+You can deploy an API Proxy to a Mule runtime.
+
+Each API proxy application receives requests on HTTP or HTTPS URLs specified by the API. Normally requests are forwarded to the corresponding API implementation endpoint URL, and then the response travels through the API proxy application back to the requesting client application.
```

Comment:
Do we need every word in "API implementation endpoint URL"?

Reply:
I would leave it at API implementation

Reply:
Done! Thanks!